gwswlib/hydx.py

```diff
@@ -0,0 +1,242 @@
+# -*- coding: utf-8 -*-
+import logging
+
+logger = logging.getLogger(__name__)
+
+
+class Hydx:
+    def __init__(self):
+        self.connection_nodes = []
+
+
+class ConnectionNode:
+    MAAIVELDSCHEMATISERINGCOLL = [
+        {"Label": "Reservoir", "Code": "RES"},
```

Comment:
Ik vermoed dat "label" en "code" niet nodig zijn? Een dict is ook goed:

    {"RES": "Reservoir", ....}

Vaak (o.a. django) is het een lijstje van lijstjes:

    [["RES", "Reservoir"], ...]

Los daarvan zou ik, tenzij het ècht nodig is, "label" en "code" zonder hoofdletter schrijven.